api/src/pcapi/core/offerers/repository.py

```diff
@@ -562,14 +563,28 @@ def get_offerer_and_extradata(offerer_id: int) -> models.Offerer | None:
         .correlate(models.Offerer)
         .exists()
     )
-
+    has_account_with_pending_corrections_subquery = (
+        sqla.select(1)
+        .select_from(finance_models.BankAccount)
+        .where(
+            sqla.and_(
+                finance_models.BankAccount.offererId == models.Offerer.id,
+                finance_models.BankAccount.isActive.is_(True),
+                finance_models.BankAccount.status
+                == finance_models.BankAccountApplicationStatus.WITH_PENDING_CORRECTIONS,
+            ),
+        )
+        .correlate(models.Offerer)
+        .exists()
+    )
     return (
         db.session.query(
             models.Offerer,
             sqla.or_(has_non_free_offers_subquery, has_non_free_collective_offers_subquery).label("hasNonFreeOffer"),
             has_valid_bank_account_subquery.label("hasValidBankAccount"),
             has_pending_bank_account_subquery.label("hasPendingBankAccount"),
             has_active_offers_subquery.label("hasActiveOffer"),
+            has_account_with_pending_corrections_subquery.label("hasBankAccountWithPendingCorrections"),
```

Comment:
[micro-détail] Par homogénéité, on pourrait nommer l'argument comme le label : 

```suggestion
            has_bank_account_with_pending_corrections_subquery.label("hasBankAccountWithPendingCorrections"),
```

... mais je survivrai. :)